swift-ticketing/test/swift_ticketing/factory.clj

```diff
@@ -49,6 +49,16 @@
    "from" (random-date)
    "to" (random-date)})
 
+(defn events-result []
+  (let [mk-event
```

Comment:
The `events-result` seems like a redundant fn. You can define `mk-event` as a fn and the callers can map over it?

Reply:
Is it still a good idea to only expose function that works on a single object if I'm only going to use the one that produces a collection? In this case, I only need the collection of `events` and don't have any need to generate a single `event`.